grakn-test/src/test/java/ai/grakn/test/graql/analytics/ScalingTestIT.java

```diff
@@ -301,7 +304,8 @@ public void testLargeDegreeMutationResultsInReadableGraphIT() throws Exception {
         addEdges(keyspace, MAX_SIZE);
 
         Grakn.factory(Grakn.DEFAULT_URI, keyspace).getGraph().showImplicitConcepts(true);
-        Grakn.factory(Grakn.DEFAULT_URI, keyspace).getGraph().graql().compute().degree().persist().execute();
+        //TODO: Remove persistence
```

Comment:
Why is there a todo here for removing persistence? Isn't this PR doing just that?